test/test_catalyst/test_langchain_tracing.py

```
@@ -0,0 +1,177 @@
+            'response': 'response', 
+            'context': 'context',
+        }
+        try:
+            evaluation.add_metrics(
+                metrics=[
+                    {'name': 'Hallucination', 'config': {'model': 'gpt-4o-mini', 'provider': 'openai', 'threshold': {'gte': 0.3}}, 'column_name': 'Hallucination_v1', 'schema_mapping': schema_mapping},
+                ]
+            )
+        except ValueError:
+            evaluation.append_metrics('Hallucination_v1')
+        except Exception as e:
```

Ruff (0.8.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)